source/features/github-bugs.css

```diff
@@ -40,19 +40,6 @@
 	top: 0;
 }
 
-/* Fix alignment of "Allow edits by maintainers" checkbox label and icons #5510 */
-#partial-discussion-sidebar .discussion-sidebar-item:last-child > .d-inline-flex {
-	align-items: center !important;
-}
-
-#partial-discussion-sidebar :is(.js-collab-option input, .js-collab-form + details summary) {
-	margin-top: 0 !important;
-}
-
-#partial-discussion-sidebar .js-collab-form .status-indicator .octicon {
-	vertical-align: baseline !important;
-}
```

Comment:
CSS no longer applied. The native alignment looks good enough to me already:

<img width="229" alt="Screenshot 4" src="https://github.com/refined-github/refined-github/assets/1402241/42b1dcce-80a6-42c7-a663-44f41e84c1c6">